Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+
+namespace Comparison_Engine.Base_Classes
+{
+    //Structs for Bar and Drink Object properties
+    public struct BarStruct
+    {
+        public string barName;
+
+        public string barLocation;
+
+        public BarStruct(string barName = null, string barLocation = null)
+        {
+            this.barName = barName;
+            this.barLocation = barLocation;
+        }
```

Comment:
why would we have default values as null?

Reply:
Just for a simple way to know if we are changing those values, as can be seen in Bar/Drink.cs

Reply:
Oh, now I think I got it, completely missed it when reviewing.